testing/ase/test_sw_c_ase.cpp

```diff
@@ -542,3 +490,34 @@ TEST(LibopaecAseOps, ase_app_01) {
 	remove_existing_lock_file("app_test2.txt");
 }
 
+/**
+* @test       ase_app_02
+*
+* @brief      When the parameters are nullptr,
+*             is_directory function returns 0;
+*
+*/
+TEST(sim_sw_ase, ase_app_02) {
+	EXPECT_EQ(0, is_directory(nullptr));
+}
+
+/**
+* @test       ase_app_03
+*
+* @brief      When the parameters of va is bigger than 48
```

Comment:
Can you expand a bit more.  Why does passing 48 cause it to fail?  Also, the description currently says "bigger than 48", but you're passing 48.

Reply:
Will update comment